lib/verifier.py

```diff
@@ -167,10 +167,35 @@ def _raise_if_valid_tx(cls, raw_tx: str):
         tx = Transaction(raw_tx)
         try:
             tx.deserialize()
-        except:
-            pass
-        else:
-            raise InnerNodeOfSpvProofIsValidTx()
+        except SerializationError:
```

Comment:
Ugh. Do you trust their crappy deserialization code not not raise random Exception somewhere? :)  I think it's ok here just to catch anything like before.

I do like the latter stuff you did.

Reply:
Haha OK :-)